dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
Can't you import cross-storage outside of your custom module loader..? You're trying to place it in the global scope anyway.

In any case, this situation def seems very unique to your current setup, but it sounds like you've got a solution with forking anyway. Best of luck! :)